03_bag.ipynb

```diff
@@ -478,32 +447,32 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": false,
     "scrolled": true
    },
    "outputs": [],
    "source": [
     "%%time\n",
-    "\n",
     "# Warning, this one takes a while...\n",
-    "print(js.groupby(lambda item: item['name']).map(lambda k, v: (k, len(v))).compute())"
+    "result = js.groupby(lambda item: item['name']).starmap(lambda k, v: (k, len(v))).compute()\n",
+    "print(sorted(result))"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "%%time\n",
-    "\n",
     "# This one is comparatively fast and produces the same result.\n",
-    "js.foldby(key='name', binop=lambda total, x: total + 1, \n",
-    "                      initial=0, \n",
-    "                      combine=lambda a, b: a + b, \n",
-    "                      combine_initial=0).compute()"
+    "from operator import add\n",
+    "incr = lambda tot, _: tot+1\n",
```

Comment:
Yes, a real `def` would be better. I was being too cute. Let me re-read the explanation when I'm not on my tablet on a plane. I think I remember it could be improved.